### PR TITLE
Docs: Fix Helm Chart links

### PR DESCRIPTION
**Pull request by saerosV**
**What this PR does / why we need it**:

- Fix the link pointing to a Loki Helm chart, found in the [Contribute to Helm](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md#contribute-to-helm) section of the [CONTRIBUTING.MD](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) file.
- Fix the link pointing to a Promtail Helm chart, found in the [Using Helm to deploy on Kubernetes](https://github.com/grafana/loki/blob/main/production/README.md#using-helm-to-deploy-on-kubernetes) section of the [production/README.md](https://github.com/grafana/loki/tree/main/production) file.

The previous links used were pointing to a [deprecated file](https://github.com/grafana/loki/tree/main/production/helm).

**Special notes for your reviewer**:

None.

**Checklist**
- [X] Documentation added
- [ ] Tests updated
- [ ] Is this an important fix or new feature? Add an entry in the `CHANGELOG.md`.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Notice: `go get` return `package github.com/grafana/loki: no Go files in /go/src
 
 ## Contribute to helm
 
-Please follow the [Loki Helm Chart](./production/helm/README.md).
+Please follow the [Loki Helm Chart](https://github.com/grafana/helm-charts/blob/main/charts/loki/README.md#loki-helm-chart).
```
Comment (trevorwhitney) [inline CONTRIBUTING.md:24]:
```suggestion
Please follow the [Loki Helm Chart](./production/helm/loki/README.md).
```

File: production/README.md [hunk 2]
```diff
@@ -52,7 +52,9 @@ orchestrator from HashiCorp.
 
 ## Using Helm to deploy on Kubernetes
 
-There is a [Helm chart](helm) to deploy Loki and Promtail to Kubernetes.
+Here are the Helm charts used to deploy Loki and Promtail to Kubernetes:
+- [Loki](https://github.com/grafana/helm-charts/blob/main/charts/loki/README.md#loki-helm-chart)
```
Comment (trevorwhitney) [inline production/README.md:56]:
```suggestion
- [Loki](./production/helm/loki/README.md) 
```

**Review by trevorwhitney — CHANGES_REQUESTED** [review]
Given the recent migration of the helm chart back into this repo, I still think these links need to be updated, but to slightly different locations.

**Comment by saerosV** [comment]
@trevorwhitney Just updated the links, as per your suggestions. I think the PR is ready to be merged now. 

Thanks for the review!

**Review by trevorwhitney — APPROVED** [review]
lgtm, thanks for the fix!

**Pull request merged**